utils/test_utils.go

```diff
@@ -88,3 +90,18 @@ func GetTestSubstate(encoding string) *substate.Substate {
 	}
 	return ss
 }
+
+// CreateTestSubstateDb creates a test substate database with a predefined substate.
+func CreateTestSubstateDb(t *testing.T) (substateDb.SubstateDB, *substate.Substate, string) {
+	path := t.TempDir()
```

Comment:
I think It is better to provide `static proved test data` for cmd test.
Main reason is because we can't be sure that SubstateDB methods work correctly all the time.